l5kit/l5kit/visualization/utils.py

```diff
@@ -7,13 +7,11 @@
 
 PREDICTED_POINTS_COLOR = (0, 255, 255)
 TARGET_POINTS_COLOR = (255, 0, 255)
-REFERENCE_TRAJECTORY_POINT_COLOR = (255, 255, 0)
+REFERENCE_TRAJ_COLOR = (255, 255, 0)
```

Comment:
Minor, but I find this a bit consisitent: It is predicted_POINTS_color and thus Id also prefer the old version actually.

Reply:
It was changed by @yye-lyft, I'm fine with both as long as there is `COLOR` at the end :)

Reply:
hmm, for me Trajectory is already means a sequence of waypoints, so I am remove `POINT`. Sorry for piggypack this change in this PR. I will create new PR for this.